webapp/app/js/graphs.js

```diff
@@ -432,6 +432,8 @@ perf.controller('GraphsCtrl', [
       }, {location: true, inherit: true,
           relative: $state.$current,
           notify: false});
+      window.document.title = ($scope.seriesList[0].name + " " + $scope.seriesList[0].platform + 
+                          " (" + $scope.seriesList[0].projectName + ")");
```

Comment:
Actually one small request here. Could you add some code to check if the length of $scope.seriesList is greater than 1, and if so, append "and others" to window.document.title?